source/reporters/ut_xunit_reporter.tpb

```diff
@@ -23,115 +23,14 @@ create or replace type body ut_xunit_reporter is
   end;
 
   overriding member procedure after_calling_run(self in out nocopy ut_xunit_reporter, a_run in ut_run) is
```

Comment:
No need for override - in it's not overridden it will call the `after_calling_run` from parent type.